packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -179,9 +179,8 @@ export class OdspSummaryUploadManager {
 		let blobs = 0;
 		const keys = Object.keys(tree.tree);
 		for (const key of keys) {
-			assert(!key.includes("/"), 0x9cd /* id should not include slashes */);
-			// Non null asserting for now, this should be changed to Object.entries
-			const summaryObject = tree.tree[key]!;
+			assert(!key.includes("/"), "id should not include slashes");
```

Comment:
why is this assert modified?

Reply:
Thank you for the catch, the revert was corrupted, I redid the revert